### PR TITLE
Fix for #237 the print command doesn't open the console

### PR DESCRIPTION
**Pull request by atgeflu**
Hi,
This should be a fix for issue #237. 

In this issue a fix was described already. I simply implemented it. 

Now the console becomes visible when using print as well as println. I couldn't think of a way to add tests for it, so there aren't any. If you can think of a way to test this, I'd be willing to try to implement it. 

Running node test didn't generate processing.min.js, so it's not included. 

processing.js contains some unrelated changes again. I guess it might be because some files were changed after processing.js was updated. 

Cheers,
Jean-Noël

### CODE REVIEW

File: src/Helpers/PjsConsole.js [hunk 1]
```diff
@@ -117,19 +122,11 @@ module.exports = function PjsConsole(document) {
...
-    }
     var args = Array.prototype.slice.call(arguments);
     args.push('<br>');
     e.print.apply(e, args);
-    if (e.BufferArray.length > e.BufferMax) {
-      e.BufferArray.splice(0, 1);
-    } else {
-      e.javaconsole.scrollTop = e.javaconsole.scrollHeight;
-    }
+    if (e.BufferArray.length > e.BufferMax) e.BufferArray.splice(0, 1);
```
Comment (Pomax) [inline src/Helpers/PjsConsole.js]: please don't include formatting changes in a functional PR, especially in this case: curly brackets serve a purpose, don't slow down code, and make things easier to read. Removing them is not a good idea =)

**Comment by Pomax** [comment]
Looks good, but remember to focus on the issue at hand: if you're adding a PR that fixes `print()`s console showing, don't also sneak in formatting changes =)

File: src/Helpers/PjsConsole.js (hunk 1 above)
Comment (GoToLoop) [inline src/Helpers/PjsConsole.js]: You don't like Python for sure. Boy, let's get rid of those annoying curly places whenever we can!  :P

**Comment by GoToLoop** [comment]
Dunno much about how **print()**'s implementation. The actual **console.log()** always append a linefeed at the end. Which of course doesn't match **print()**'s expected behavior...

**Comment by Pomax** [comment]
Note that the `print` and `println` functions do not rely on the `window.console` API, they write string data into an overlay div, so `console.log` behaviour is not be a concern.

**Comment by GoToLoop** [comment]
Alas, censorship again as always...

**Comment by atgeflu** [comment]
I undid the formatting changes. I only did them because the original poster thought it might be good to keep the formatting consistent throughout the file ;-)

But thanks for the feedback, I'll keep this in mind for future pull request.

**Comment by GoToLoop** [comment]
In my deleted post, I've agreed w/ your 1 liner `if ()`. It's tough to contribute here...

**Comment by atgeflu** [comment]
I personally don't mind the curly braces. 
Mike's got a point though, not to mix formatting changes with the fix. Might also make it easier to check changes later in case something broke for example. 
So I'll happily apply his advice and keep PRs focused on the actual fixes.

**Comment by Pomax** [comment]
@GoToLoop apologies for that: I'm deleting anything that sounds mocking rather than on target for a PR (your comment was on python style, which has nothing to do with a JS)

**Pull request merged**